src/data/faq_de.json

```diff
@@ -50,11 +120,70 @@
+                                "anchor": "ramp_down_tests",
+                                "active": false,
+                                "textblock": [
+                                    "Dies wird nur noch bis 19. April 2023 möglich sein, wenn die Infrastruktur abgeschaltet wird. Bitte erkundigen Sie sich für den Abruf von Testergebnissen und Laborbefunden bei dem jeweiligen Anbieter."
+                                ]
+                            },
+                            {
+                                "title": "Kann ich nach einem positiven Test weiterhin Andere über die CWA warnen?",
+                                "anchor": "ramp_down_warn",
+                                "active": false,
+                                "textblock": [
+                                    "Sie können andere CWA-Nutzende noch bis einschließlich 30. April 2023 über ein positives Testergebnis informieren und diese damit warnen."
+                                ]
+                            },
+                            {
+                                "title": "Kann ich für die Corona-Warn-App noch Event-QR-Codes ausstellen?",
+                                "anchor": "ramp_down_qr_codes",
+                                "active": false,
+                                "textblock": [
+                                    "Eine Generierung neuer QR-Codes wird innerhalb der App bis einschließlich 30. April 2023 möglich sein."
+                                ]
+                            },
+                            {
+                                "title": "Kann ich bei einem Event-QR-Code noch einchecken und Andere warnen?",
+                                "anchor": "ramp_down_check_in",
+                                "active": false,
+                                "textblock": [
+                                    "Nein, ein Warnen über die Event-QR-Codes ist aufgrund der fehlenden Infrastruktur nicht mehr möglich. Daher wird die App diese Funktion im Release 3.2 ab 1. Mai 2023 auch nicht mehr anbieten."
+                                ]
+                            },
+                            {
+                                "title": "Was ist, wenn die App Störungen/Fehler hat; werden diese dann nicht mehr beseitigt? Kann ich diese weiterhin melden?",
+                                "anchor": "ramp_down_reports",
+                                "active": false,
+                                "textblock": [
+                                    "Bis einschließlich 30. April.2023 können aus der App heraus noch Fehlerberichte an die Entwicklung übertragen werden. Danach ist es zwar möglich, eventuelle Fehler und Störungen bis einschließlich 31. Mai 2023 zu melden. Jedoch können Bugfixes zur noch für solche Fehler erstellt werden, die bis einschließlich 15. Mai 2023 gemeldet wurden. Entsprechend ist es auch in GitHub ab 16. Mai 2023 nicht mehr möglich, neue Issues zu eröffnen."

```

Comment:
```suggestion
                                    "Bis einschließlich 30. April. 2023 können aus der App heraus noch Fehlerberichte an das Entwicklungs-Team übertragen werden. Danach ist es zwar möglich, eventuelle Fehler und Störungen bis einschließlich 31. Mai 2023 zu melden. Jedoch können Bugfixes zur noch für solche Fehler erstellt werden, die bis einschließlich 15. Mai 2023 gemeldet wurden. Entsprechend ist es auch auf GitHub ab dem 16. Mai 2023 nicht mehr möglich, neue Issues zu öffnen."
```

Reply:
"zur noch" => "nur noch"

Reply:
Herzlichen Dank für das aufmerksame Lesen!

Reply:
Updated in https://github.com/corona-warn-app/cwa-website/pull/3435/commits/73c82ad32454368aa9de62973567f834a1c8e0ca, thanks!